x/auth/types/keys.go

```diff
@@ -0,0 +1,30 @@
+package types
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+const (
+	// module name
+	ModuleName = "auth"
+
+	// StoreKey is string representation of the store key for auth
+	StoreKey = "acc"
+
+	// FeeStoreKey is a string representation of the store key for fees
+	FeeStoreKey = "fee"
+
+	// QuerierRoute is the querier route for acc
+	QuerierRoute = StoreKey
+)
+
+var (
+	// AddressStoreKeyPrefix prefix for account-by-address store
+	AddressStoreKeyPrefix = []byte{0x01}
```

Comment:
Is this name standard for each module/store? Is this something we could add to the module interface? Is this even a good question to ask?

Reply:
Each module has completely independant store needs - and thus key needs for it's store, this would not make sense to generalize, I don't even know how one would! (but not a bad question!)